eos/form-factors/parametric-kkvdz2022.hh

```diff
@@ -0,0 +1,90 @@
+/* vim: set sw=4 sts=4 et foldmethod=syntax : */
+
+/*
+ * Copyright (c) 2022 Danny van Dyk
+ * Copyright (c) 2022 Stephan Kuerten
+ *
+ * This file is part of the EOS project. EOS is free software;
+ * you can redistribute it and/or modify it under the terms of the GNU General
+ * Public License version 2, as published by the Free Software Foundation.
+ *
+ * EOS is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program; if not, write to the Free Software Foundation, Inc., 59 Temple
+ * Place, Suite 330, Boston, MA  02111-1307  USA
+ */
+
+#ifndef EOS_GUARD_SRC_FORM_FACTORS_PARAMETRIC_KKVDZ2022_HH
+#define EOS_GUARD_SRC_FORM_FACTORS_PARAMETRIC_KKVDZ2022_HH 1
+
+#include <eos/form-factors/mesonic.hh>
+#include <eos/utils/options.hh>
+#include <eos/utils/parameters.hh>
+#include <eos/utils/qualified-name.hh>
+#include <eos/utils/reference-name.hh>
+
+#include <memory>
+
+namespace eos
+{
+    class KKvDZ2022FormFactors :
+        public FormFactors<PToGammaOffShell>
+    {
+        private:
+            UsedParameter _N_omega_1_0;
+            UsedParameter _N_omega_1_1;
+            UsedParameter _N_omega_1_2;
+            UsedParameter _N_omega_2_0;
+            UsedParameter _N_omega_2_1;
+            UsedParameter _N_omega_2_2;
+            UsedParameter _N_omega_3_0;
+            UsedParameter _N_omega_3_1;
+            UsedParameter _N_omega_3_2;
+            UsedParameter _N_omega_4_0;
+            UsedParameter _N_omega_4_1;
+            UsedParameter _N_omega_4_2;
+            UsedParameter _N_rho_1_0;
+            UsedParameter _N_rho_1_1;
+            UsedParameter _N_rho_1_2;
+            UsedParameter _N_rho_2_0;
+            UsedParameter _N_rho_2_1;
+            UsedParameter _N_rho_2_2;
+            UsedParameter _N_rho_3_0;
+            UsedParameter _N_rho_3_1;
+            UsedParameter _N_rho_3_2;
+            UsedParameter _N_rho_4_0;
+            UsedParameter _N_rho_4_1;
+            UsedParameter _N_rho_4_2;
+
+            double _widthOmega(const double & q2) const;
+            double _widthRho(const double & q2) const;
+            double _zOmega(const double & k2) const;
+            double _zRho(const double & k2) const;
+            double _heaviside(const double & x) const;
+
+        public:
```

Comment:
I would add
```c++
using mB = BToRho::mB;
using mrho = BToRho::mV;
using momega = BToOmega::mV;
```

Reply:
I added it in a similar way defined by the mesonic-processes.hh hope that's fine.